packages/@azure/servicebus/data-plane/test/topicFilters.spec.ts

```diff
@@ -126,28 +126,18 @@ async function sendOrders(): Promise<void> {
 }
 
 async function receiveOrders(client: SubscriptionClient): Promise<ServiceBusMessage[]> {
-  let errorFromErrorHandler: Error | undefined;
   const receivedMsgs: ServiceBusMessage[] = [];
   const receiver = client.getReceiver();
-  receiver.receive(
```

Comment:
Why receiveBatch() instead of receive() ?

Reply:
So that we can avoid having the delay().
All the delay in various tests are adding up and resulting in our tests taking a long time.
So we are going through tests and avoiding it where ever possible.

Reply:
We wanted to get rid of unnecessary delays that come with setting delays on streaming receiver.